geofence - schema not up to date

https://github.com/georchestra/georchestra/blob/18.12/postgresql/080-geofence.sql  does not seem up to date: the `styleName` column does not exist in table `gf_layer_details`.
```
08/02/2019 18:18:0508 Feb 17:18:05 ERROR [context.ContextLoader] - Context initialization failed
08/02/2019 18:18:05org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'geofenceEntityManagerFactory' defined in URL [jar:file:/tmp/jetty/jetty-0.0.0.0-8080-geoserver.war-_geoserver-any-5131545202404601747.dir/webapp/WEB-INF/lib/geofence-persistence-3.2-20180503.181323-25.jar!/applicationContext-geofenceDatasource.xml]: Invocation of init method failed; nested exception is javax.persistence.PersistenceException: [PersistenceUnit: geofence] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.MappingException: Could not determine type for: java.util.Set, at table: gf_layer_details, for columns: [org.hibernate.mapping.Column(styleName)]
08/02/2019 18:18:05	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1628)
08/02/2019 18:18:05	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:555)
08/02/2019 18:18:05	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
08/02/2019 18:18:05	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
08/02/2019 18:18:05	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
08/02/2019 18:18:05	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
08/02/2019 18:18:05	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
08/02/2019 18:18:05	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1081)
08/02/2019 18:18:05	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:856)
08/02/2019 18:18:05	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:542)
08/02/2019 18:18:05	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:443)
08/02/2019 18:18:05	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:325)
08/02/2019 18:18:05	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:107)
08/02/2019 18:18:05	at org.geoserver.platform.GeoServerContextLoaderListener.contextInitialized(GeoServerContextLoaderListener.java:22)
08/02/2019 18:18:05	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:957)
08/02/2019 18:18:05	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:552)
08/02/2019 18:18:05	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:922)
08/02/2019 18:18:05	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:364)
08/02/2019 18:18:05	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1497)
08/02/2019 18:18:05	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1459)
08/02/2019 18:18:05	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:852)
08/02/2019 18:18:05	at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:278)
08/02/2019 18:18:05	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:545)
08/02/2019 18:18:05	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
08/02/2019 18:18:05	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:138)
08/02/2019 18:18:05	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
08/02/2019 18:18:05	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:113)
08/02/2019 18:18:05	at org.eclipse.jetty.server.handler.ContextHandlerCollection.doStart(ContextHandlerCollection.java:168)
08/02/2019 18:18:05	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
08/02/2019 18:18:05	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:138)
08/02/2019 18:18:05	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
08/02/2019 18:18:05	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:113)
08/02/2019 18:18:05	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
08/02/2019 18:18:05	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:138)
08/02/2019 18:18:05	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:108)
08/02/2019 18:18:05	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:113)
08/02/2019 18:18:05	at org.eclipse.jetty.server.handler.StatisticsHandler.doStart(StatisticsHandler.java:256)
08/02/2019 18:18:05	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
08/02/2019 18:18:05	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:138)
08/02/2019 18:18:05	at org.eclipse.jetty.server.Server.start(Server.java:415)
08/02/2019 18:18:05	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:108)
08/02/2019 18:18:05	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:113)
08/02/2019 18:18:05	at org.eclipse.jetty.server.Server.doStart(Server.java:382)
08/02/2019 18:18:05	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
08/02/2019 18:18:05	at org.eclipse.jetty.xml.XmlConfiguration$1.run(XmlConfiguration.java:1572)
08/02/2019 18:18:05	at org.eclipse.jetty.xml.XmlConfiguration$1.run(XmlConfiguration.java:1512)
08/02/2019 18:18:05	at java.security.AccessController.doPrivileged(Native Method)
08/02/2019 18:18:05	at org.eclipse.jetty.xml.XmlConfiguration.main(XmlConfiguration.java:1511)
08/02/2019 18:18:05	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
08/02/2019 18:18:05	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
08/02/2019 18:18:05	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
08/02/2019 18:18:05	at java.lang.reflect.Method.invoke(Method.java:498)
08/02/2019 18:18:05	at org.eclipse.jetty.start.Main.invokeMain(Main.java:220)
08/02/2019 18:18:05	at org.eclipse.jetty.start.Main.start(Main.java:490)
08/02/2019 18:18:05	at org.eclipse.jetty.start.Main.main(Main.java:77)
08/02/2019 18:18:05Caused by: javax.persistence.PersistenceException: [PersistenceUnit: geofence] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.MappingException: Could not determine type for: java.util.Set, at table: gf_layer_details, for columns: [org.hibernate.mapping.Column(styleName)]
08/02/2019 18:18:05	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.buildNativeEntityManagerFactory(AbstractEntityManagerFactoryBean.java:396)
08/02/2019 18:18:05	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.afterPropertiesSet(AbstractEntityManagerFactoryBean.java:371)
08/02/2019 18:18:05	at org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean.afterPropertiesSet(LocalContainerEntityManagerFactoryBean.java:336)
08/02/2019 18:18:05	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1687)
08/02/2019 18:18:05	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1624)
08/02/2019 18:18:05	... 54 more
08/02/2019 18:18:05Caused by: org.hibernate.MappingException: Could not determine type for: java.util.Set, at table: gf_layer_details, for columns: [org.hibernate.mapping.Column(styleName)]
08/02/2019 18:18:05	at org.hibernate.mapping.SimpleValue.getType(SimpleValue.java:431)
08/02/2019 18:18:05	at org.hibernate.mapping.SimpleValue.isValid(SimpleValue.java:398)
08/02/2019 18:18:05	at org.hibernate.mapping.Property.isValid(Property.java:225)
08/02/2019 18:18:05	at org.hibernate.mapping.PersistentClass.validate(PersistentClass.java:595)
08/02/2019 18:18:05	at org.hibernate.mapping.RootClass.validate(RootClass.java:265)
08/02/2019 18:18:05	at org.hibernate.boot.internal.MetadataImpl.validate(MetadataImpl.java:329)
08/02/2019 18:18:05	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:443)
08/02/2019 18:18:05	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:879)
08/02/2019 18:18:05	at org.springframework.orm.jpa.vendor.SpringHibernateJpaPersistenceProvider.createContainerEntityManagerFactory(SpringHibernateJpaPersistenceProvider.java:60)
08/02/2019 18:18:05	at org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean.createNativeEntityManagerFactory(LocalContainerEntityManagerFactoryBean.java:360)
08/02/2019 18:18:05	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.buildNativeEntityManagerFactory(AbstractEntityManagerFactoryBean.java:384)
08/02/2019 18:18:05	... 58 more
```

There might be other issues

--- a/geoserver/pom.xml
+++ b/geoserver/pom.xml
@@ -18,6 +18,11 @@
     <marlin.version>0.7.5-Unsafe</marlin.version>
     <gs.spring.version>4.3.7.RELEASE</gs.spring.version>
     <server>generic</server>
+    <!-- overrides the versions provided by spring-boot in the geOrchestra root pom -->
+    <hibernate.version>3.6.0.Final</hibernate.version>
+    <spring.version>4.3.7.RELEASE</spring.version>
+    <commons-beanutils.version>1.8.0</commons-beanutils.version>
+    <commons-digester.version>1.7</commons-digester.version>
   </properties>
   <modules>
     <module>geoserver-submodule/src/web/core</module>

